insights/parsers/rpm_v_packages.py

```diff
@@ -58,3 +66,56 @@ def parse_content(self, content):
                 entry = {"line": line.strip(), "attributes": line_parts[0], "mark": None,
                          "file": line_parts[1]}
             self.packages_list.append(entry)
+
+
+@parser(Specs.rpm_V_package)
+class RpmVPackage(CommandParser):
+    """
+    Class for parsing ``/bin/rpm -V <package>`` command.
+
+    Attributes:
+        package_name (String): The name of package
+        package_list (list of dictionaries): Every dictionary contains information about one entry
+
+    Sample output of this command is::
+
+        package procps is not installed
+        ..?......  c /etc/sudoers
+        ..?......    /usr/bin/sudo
+        ..?......    /usr/bin/sudoreplay
+        missing     /var/db/sudo/lectured (Permission denied)
+
+    Examples:
+        >>> type(rpm_v_pkg)
+        <class 'insights.parsers.rpm_v_packages.RpmVPackage'>
+        >>> rpm_v_pkg.package_name
+        'coreutils'
+        >>> len(rpm_v_pkg.package_list)
+        5
+        >>> sorted(rpm_v_pkg.package_list[0].items())
+        [('attributes', None), ('file', None), ('line', 'package procps is not installed'), ('mark', None)]
+        >>> sorted(rpm_v_pkg.package_list[1].items())
+        [('attributes', '..?......'), ('file', '/etc/sudoers'), ('line', '..?......  c /etc/sudoers'), ('mark', 'c')]
+"""
+
+    def parse_content(self, content):
+        self.package_list = []
+        self.package_name = self.file_path.split("/")[-1].split('_')[-1] if self.file_path else None
+
+        for line in content:
+            line_parts = line.split()
+            if not line_parts:
+                continue
+
+            if "package" in line_parts[0] or "missing" in line_parts[0]:
```

Comment:
For `package xxxx is not installed` line, I believe we can go ahead and raise a `SkipComponent` since the package is not installed at all.